spec/controllers/admin/stripe_connect_settings_controller_spec.rb

```diff
@@ -30,7 +30,7 @@
 
       context "when a Stripe API key is not set" do
         before do
-          allow(Stripe).to receive(:api_key) { nil }
+          Stripe.api_key = nil
```

Comment:
Ah, in this case I think this setting will be persisted between tests...

Reply:
Hm, even setting it back after that spec runs still results in failures elsewhere in the build. I'm going to revert that commit for now and hopefully get the build back to green.